services/movie.py

```diff
@@ -1,11 +1,12 @@
 from django.db.models import QuerySet
-
+from django.db import transaction
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
+        title: str = None
```

Comment:
Why did you move arguments to right on 4 spaces?
```suggestion
    genres_ids: list[int] = None,
    actors_ids: list[int] = None,
    title: str = None
```

Reply:
I use ctrl + alt + L  shortcut  to format code and I thought it's good

Reply:
I always used 4 spaces.It's 1st time when I had to make it 2